content/swift/concepts/arrays/terms/reduce/reduce.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: '.reduce()'
+Description: 'Loops/Iterates over every item in a sequence, combines them into one value, and returns the combined result.'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'Arrays'
+  - 'Methods'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.reduce()`** method produces a single value from the elements of an entire sequence using the given closure.
+
+## Syntax
+
+```pseudo
+arrayName.reduce(_:_:) 
```

Comment:
Instead of using unnamed variables, can we create a sea tic name placeholder for these parameters?

Reply:
Hey @yangc95 I didn't get this point. Should we make the syntax as arrayName.reduce(initialValue,closure)?